internal/configconverter/expvar.go

```diff
@@ -57,14 +57,14 @@ func GetExpvarConverter() *ExpvarConverter {
 			instance := expvarConverterInstance
 			instance.effectiveMutex.RLock()
 			defer instance.effectiveMutex.RUnlock()
-			configYAML, _ := yaml.Marshal(instance.effective)
+			configYAML, _ := yaml.Marshal(simpleRedact(instance.effective))
```

Comment:
Is the simple redaction required to be part of this change, or is it just convenient to include with this PR? Should we be adding a second bug fix changelog entry for this change?

Reply:
e5db6ba7f70502bf1a356e0eca8470a7b3f1ab10